### ISSUE TITLE
control.capture_as_image got a low resolution image

### ISSUE THREAD

**Issue by eastam** [pywinauto/pywinauto#821]
When I use capture_as_image to save the screen_shot for the controls, I only got a low resolution image.

The code is like:
image = control1.capture_as_image()
image.show()       # _it shows a captured image correctly, but the image is blurred and seems to be a low resolution image._
image.save(path,format="PNG")   # it saved sucessfully, but with the same resolution as image.show()

How can I capture an high resolution image with control.capture_as_image()? Or what's the wrong parameter I set for the function?   I want to do some OCR for the captured image, but even with my eyes, the quality is too low. This may cause the high error rate subsequently.

**Comment by vasily-v-ryabov** [comment]
Are you running the script in a Remote Desktop session? I'm asking because RDP provides virtual desktop with lower resolution than might be used for the remote PC in native mode (through GPU).

**Comment by eastam** [comment]
No. I am running the code in local PC.

**Comment by airelil** [comment]
@eastam, did you try to play with DPI and compress_level parameters of `image.save()` as, for example, discussed here: https://stackoverflow.com/questions/40829577/setting-image-resolution-in-python-pillow ?

**Comment by eastam** [comment]
> @eastam, did you try to play with DPI and compress_level parameters of `image.save()` as, for example, discussed here: https://stackoverflow.com/questions/40829577/setting-image-resolution-in-python-pillow ?

em. A good suggestion. I will have a look on the resolution setting of the image.save() to see whether it will improve. But still one question for me is that actually I do the image.show() without saving image, it also show a low resolution image. I wonder whether there is some parameter setting for the function capture_as_image(). Are there also DPI setting for image.show()? I will also check this. Thanks.

**Comment by airelil** [comment]
Generally, pywinauto uses `PIL.ImageGrab.grab()` method, providing to it a tuple with the control coordinates. Could you pls check the output of `control1.rectangle()` so we can see the size of the grabbed box.

**Comment by eastam** [comment]
I found maybe it is the issue of default image display software for win10. When I call the image.show(), it  actually open the win10 image display software to show the image. The display is not everytime with correct resolution. The same happened when I dobule click the saved image file to view it. When I switched to other third party image viewer, I found the image is in good resolution even during the image.show() it is with low resolution effect. 

So, it is not the issue of the capture_as_image(), it is the issue of the win10 default image viewer to display the image in a wrong way. Thanks all for your help!
